test/lsp.py

```diff
@@ -948,6 +948,11 @@ def setup_lsp(
                 params['initializationOptions'] = {}
             params['initializationOptions']['include-paths'] = custom_include_paths
 
+        if self.trace_io:
+            if params['initializationOptions'] is None:
+                params['initializationOptions'] = {}
+            params['initializationOptions']['trace-log-file'] = "solc.log"
```

Comment:
Would be nice if was possible to immediately tell at a glance that this is specifically the log from the LSP and not something else `solc` might do:
```suggestion
            params['initializationOptions']['trace-log-file'] = "solc-lsp.log"
```